### ISSUE TITLE
Can't add Fibaro Wall Plug

### ISSUE THREAD

**Issue by ocatie** [athombv/homey#410]
With 0.8.27 the interviewing process for the Fibaro Wall Plug still fails. It does add an on/off switch, but this only works until Homey is rebooted. After a reboot, the device shows a broken connection link with 
error_timeout_waiting_for_report
The 'broken link' makes that you cannot even delete the device, as that button is blocked (same as for device settings).

https://cloud.githubusercontent.com/assets/11725674/14227764/3d0c1e4e-f905-11e5-9840-fdb525e91819.png

**Comment by ocatie** [comment]
@nlrb Could you please retry to add the device, send the log and past the given code here?

**Comment by Muyz** [comment]
Same holds for Fibaro dimmer.

**Comment by ocatie** [comment]
Please create a separate issue for a different device. So we can keep log id's etc organised.

**Comment by djeskoswe** [comment]
My code after fibaro wall plug fail F16CD1E33D

**Comment by bliekp** [comment]
Adding the plug worked fine on 0.8.29. I could also use it. However, after several hours the connection is lost in the same way as described above. Code for crashlog: 4F5CCC0C5F.

**Comment by ocatie** [comment]
In both logs the same get command isn't answered. Could be a error in the get command. What version of the plug are you both using?

**Comment by bliekp** [comment]
How can I check the version of the plug?

**Comment by ocatie** [comment]
In the device manual.

**Comment by bliekp** [comment]
Ouch... not sure I still have that...

**Comment by freemann** [comment]
As far as i known there only "one" version and the manuals described it as "FGWPE/F-101 v.2.1 - v.2.3."  This version has the basic firmware 22.22 and supports these Z-Wave classes:

Supported Command Classes
- Basic (version 1)
- Association (version 2)
- Version (version 1)
- Alarm (version 0)
- Manufacturer Specific (version 1)
- Configuration (version 1)
- Multilevel Sensor (version 2)
- Meter (version 2)
- Powerlevel (version 1)
- Firmware Update Meta Data (version 1)
- Alarm Sensor (version 0)
- Binary Switch (version 1)

**Comment by bliekp** [comment]
Also, it never reported kWh or W....

**Comment by freemann** [comment]
If i'm right i read somewhere that Z-Wave version 2 is not (fully) implemented yet? 
If so, that could explain why "Meter"(ing) is not working yet ?

**Comment by bliekp** [comment]
It worked fine on my Vera 3.

**Comment by nlrb** [comment]
My wall plug version is 2.2.

**Comment by freemann** [comment]
@nlrb 
How did you identify your plug? I also have a multiple of these in my house and can't find anything (anymore) on them to identify it.

**Comment by nlrb** [comment]
@freemann there is a sticker on the box the plug came in. I didn't throw out the boxes luckily :-).

**Comment by freemann** [comment]
Oops.... My devices are >3 years old. Storing all Domotica related box in a closed would fill a Billy.

Bit of a shame to Fibaro.

**Comment by ocatie** [comment]
I didn't asked it because of different supported command classes. Please ask questions on the forum or use DM. This way we can keep the information/comment ratio on github as high as possible.

**Comment by freemann** [comment]
You asked for a version, we try to find the version, you said manual, its the box, now everyone concerning this issue knows where to look for. 

To me all this is issue related :smiley:

**Comment by Frakke0** [comment]
My Fibaro wall plugs are version: FGWPE/F-102

**Comment by djeskoswe** [comment]
timeout error after reboot fibaro plug.  log 57D0441819   FW = .30

**Comment by freemann** [comment]
This morning tried a FGWPE/F-101 and after about 30 second "1/1 interviewing" i got the message:

> Your Z-Wave device could not be interviewed. The device has been added as a basic device.

The device came out of my Vera3 network, so I first reset it with "settings>Zwave>remove node" with a positive result and then did a reset.

This is my first device which isn't working.....

Looked for the box to see which version it is, only could found the manual.

Here's a Log;
**A839259E44**

**Comment by ocatie** [comment]
I'm contacting Fibaro about this one. I'll let you know when I know more.

**Comment by freemann** [comment]
Yesterday I moved the Fibaro Wall plug to the Attic and Homey refuses to refresh it.
I tried to refresh but didn't help, then did tried a Zwave chip reset and that also didn't work.
Then I did a full reboot of Homey and again tried to refresh.
1 hour after the reboot the Wallplug is still dead.

Here some system logs:
Before the reboot: 4BC1954CE2
After the reboot:   02C22BDA4D
After the refresh:  9D768C9F88

**Comment by jghaanstra** [comment]
Argh, really thought this one would work with Homey as it's sold a lot. Should have done my research first. I added it with the same result as mentioned above (.. has been added as a basic on/off device). All it did was indeed switch on and off. So I decided to remove the node so I could try adding it again. Didnt work. The socket was excluded but the node remained available in Homey but always with the status "Error: timeout waiting for report". Refreshing the node does nothing, also not mark it as failed so I cant remove it either. My version is FGWPE/F-101 v.2.1 - v.2.3. I have attached the manual and send a log, perhaps it helps.

Log: A75D6BC420

[Fibaro FGWPE_F-101-Wall-Plug-en-2.1-2.3.pdf](https://github.com/athombv/homey/files/259850/Fibaro.FGWPE_F-101-Wall-Plug-en-2.1-2.3.pdf)
